docs/UserGuide.md

```diff
@@ -573,13 +579,13 @@ coming soon
+| **Edit an individual event**                  | `eedit EVENT_INDEX [r/ROLE_WITH_QUANTITY]... [sd/START_DATE_AND_TIME] [ed/END_DATE_AND_TIME] [l/LOCATION] [dsc/DESCRIPTION] [m/MATERIAL_WITH_QUANTITY]... [b/BUDGET] [vs/MAX_VOLUNTEER_COUNT]` <br> e.g, `eedit 1 r/10 farmer ed/20/11/2024 1300 l/malaysia vs/3`                                                                       |
 | **Read an individual event**                  | `eshow EVENT_INDEX` <br> e.g., `eshow 8`                                                                                                                                                                                                                                                                                                |
 | **Delete an event**                           | `edelete EVENT_INDEX` <br> e.g., `edelete 3`                                                                                                                                                                                                                                                                                            |
-| **Add a quantity of materials to an event**   | `eaddm eid/EVENT_INDEX m/MATERIALS_AND_LOGISTICS_NEEDED`<br> e.g., `eaddm eid/1 m/10 potato`                                                                                                                                                                                                                                            |
-| **Create a new volunteer profile**            | `vcreate n/VOLUNTEER_NAME p/PHONE_NUMBER e/EMAIL [s/SKILLS]...`<br> e.g.,`vcreate n/John Lim p/81234567 e/john123@gmail.com s/Cooking`                                                                                                                                                                                                  |
+| **Add a quantity of materials to an event**   | `eaddm eid/EVENT_INDEX m/MATERIAL_WITH_QUANTITY`<br> e.g., `eaddm eid/1 m/10 potato`                                                                                                                                                                                                                                                    |
+| **Create a new volunteer profile**            | `vcreate n/NAME p/PHONE e/EMAIL [s/SKILL]...`<br> e.g.,`vcreate vn/John Lim hp/81234567 e/john123@gmail.com s/Cooking`                                                                                                                                                                                                                  |
```

Comment:
oops theres a `vn/` and `hp` in the example

Reply:
lol can i resolved a merge conflict then that happened

Reply:
OKI